substrate/frame/broker/src/lib.rs

```diff
@@ -390,6 +390,14 @@ pub mod pallet {
 			/// The Region whose contribution is no longer exists.
 			region_id: RegionId,
 		},
+		/// A region has been force-removed from the pool. This is usually due to a provisionally
+		/// pooled region being redeployed.
+		RegionUnpooled {
```

Comment:
This is the reason for the major bump, but since there is an event for a region being pooled we should also have one for it being unpooled.